custom/icds_reports/templates/icds_reports/dashboard.html

```diff
@@ -57,8 +57,8 @@
         <script src="{% static 'js/topojsons/districts_v2.topojson.js' %}"></script>
         <script src="{% static 'js/topojsons/blocks_v3.topojson.js' %}"></script>
     {% else %}
-        <script src="{% static 'js/topojsons/states.topojson.js' %}"></script>
-        <script src="{% static 'js/topojsons/district.topojson.js' %}"></script>
+        <script src="{% static 'js/topojsons/states_v2.topojson.js' %}"></script>
```

Comment:
These are the same with and without feature flags `have_access_to_features` so they can be removed from the if/else statement.

If I remember correctly this was done because the size of districts_v2 & states_v2 is much greater and we were unsure of the performance on MWCD computers. Was that ever tested?

Also why isn't blocks updated to v3?

Reply:
@emord 
I didn't notice that we have the same files hidden by feature flag.

But now on the production, we have all states, districts, blocks, sectors and awcs added and in files which are not hidden by feature flag, we don't have a configuration to show them. 

This was added very long ago so probably no one tested this. I reverted changes in the dashboard.